### PR TITLE
Copy smoke tests from external repo

### PR DESCRIPTION
**Pull request by frankieroberto**
This just copies across the "smoke tests" (written using [cypress](https://www.cypress.io)) from the external [find-teacher-training-tests](https://github.com/DFE-Digital/find-teacher-training-tests) repository.

This should enable us to keep the tests more in sync with the code, for example updating both in a single pull request.

### CODE REVIEW

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -21,10 +21,6 @@ jobs:
 
       - name: Checkout
```
Comment (duncanjbrown) [inline .github/workflows/smoke-test.yml:22]: Could we lose this bit too?

File: .github/workflows/smoke-test.yml [hunk 2]
```diff
@@ -54,7 +50,7 @@ jobs:
           if-no-files-found: ignore
           retention-days: 7
 
-      - name: 'Nofiy #twd_apply_tech on failure'
+      - name: 'Notify #twd_apply_tech on failure'
```
Comment (duncanjbrown) [inline .github/workflows/smoke-test.yml:53]: ⭐

File: .github/workflows/smoke-test.yml (hunk 1 above)
Comment (frankieroberto) [reply .github/workflows/smoke-test.yml:22]: I was assuming it'd still need to checkout the repo, in order to get the actual tests?

Comment (duncanjbrown) [reply .github/workflows/smoke-test.yml:22]: ah i see — sorry, thought this was the action for the main bit, ignore me!

**Comment by frankieroberto** [comment]
@duncanjbrown let's merge this, and do a follow-up PR which updates the tests and turns on the feature flag by default?

**Review by duncanjbrown — APPROVED** [review]

**Pull request merged**